daemons/execd/remoted_tls.c

```diff
@@ -131,6 +133,12 @@ lrmd_remote_client_msg(gpointer data)
         crm_xml_add_int(request, F_LRMD_CALLID, lrmd_call_id);
 
         process_lrmd_message(client, id, request);
+
+        /* If this was a register operation, also ask for new schema files. */
+        if (pcmk__str_eq(op, CRM_OP_REGISTER, pcmk__str_none)) {
+            remoted_request_cib_schema_files();
+        }
+
```

Comment:
I think it makes sense to do this in process_lrmd_signon() after ipc_proxy_add_provider()